src/components/FlowRunStateTypeTabs.vue

```diff
@@ -0,0 +1,71 @@
+<template>
+  <p-tabs v-model:selected="selected" :tabs="tabs" class="flow-run-state-type-tabs">
+    <template #heading="{ tab }">
+      <template v-if="tab">
```

Comment:
Is this `v-if` check necessary? It doesn't seem like `tab` can be undefined from the type

Reply:
Something about the slot types for the heading slot makes tab possibly undefined. Briefly looked into it but wasn't 5 minute fix so decided using a v-if as a type guard was the simple fix. More growing pains with slot types appearing that previously were not available.

Reply:
😞 🤠